conanfile.py

```diff
@@ -6,7 +6,7 @@ class MeshlinklibConan(ConanFile):
     license = "<Put the package license here>"
```

Comment:
We use 2 spaces as tabs everywhere. We should use the same convention here too.

Reply:
this is .py file so if i give 2 spaces its showing syntax  error

Reply:
It will not give error. But let's confirm with Manav tomorrow whether we should make this change or not.

Reply:
its giving error
ERROR: Error loading conanfile at '/home/preetimuttagi/conan-dir/mesh/meshlink/conanfile.py': Unable to load conanfile in /home/preetimuttagi/conan-dir/mesh/meshlink/conanfile.py
  File "/usr/lib/python3.5/imp.py", line 172, in load_source
    module = _load(spec)
  File "<frozen importlib._bootstrap>", line 693, in _load
  File "<frozen importlib._bootstrap>", line 673, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 661, in exec_module
  File "<frozen importlib._bootstrap_external>", line 767, in get_code
  File "<frozen importlib._bootstrap_external>", line 727, in source_to_code
  File "<frozen importlib._bootstrap>", line 222, in _call_with_frames_removed
  File "/home/preetimuttagi/conan-dir/mesh/meshlink/conanfile.py", line 16
    def build(self):
    ^
IndentationError: unexpected indent